backtracking/parentheses.py

```diff
@@ -0,0 +1,19 @@
+def generateParenthesis(n):
```

Comment:
you soluld follow naming convention. ex: def generate_parenthesis(number:int ) -> str:

define paramter type and return too.